internal/pkg/cli/pipeline_update.go

```diff
@@ -159,7 +162,7 @@ func (o *updatePipelineOpts) shouldUpdate() (bool, error) {
 	return shouldUpdate, nil
 }
 
-type codestar interface {
+type codeStar interface {
```

Comment:
We don't need to define this interface at the package level! It can be anonymous, for example: https://play.golang.org/p/536QYoQLIx8 so:

```go
_, ok := in.Source.(interface {
     ConnectionName() (string, error)
})
```